example/App.tsx

```diff
@@ -32,27 +32,209 @@ function App(): React.JSX.Element {
           }}
         >
           The dropdown menu will display above the input box when there
-          isn&quot;t enough space below
+          isn&apos;t enough space below
         </Text>
         <Select
           data={data}
-          onSelect={(e: Data) => {
-            console.log(e.label);
-          }}
+          onSelect={console.log}
```

Comment:
Do we want `console.log()` as the `onSelect`? This same question goes for all the `onSelect`s in the example app

Reply:
I used `console.log` because I didn't want to add too much unnecessary complexity to the example app, since it's mainly for showcasing and testing. After taking a second look, I do agree that `console.log` by itself can be ambiguous and maybe even lazy. I've modified it further so that every Select and MultiSelect would have a `onDataSelect` or `onMultiDataSelect`, hopefully making it more clear on what it's doing.